core/src/main/scala/com/databricks/labs/remorph/intermediate/extensions.scala

```diff
@@ -233,3 +233,4 @@ case class KnownInterval(value: Expression, iType: KnownIntervalType) extends Ex
   override def children: Seq[Expression] = Seq(value)
   override def dataType: DataType = UnresolvedType
 }
+case class ColumnDetail(name: String, dataType: DataType, nullable: Boolean = true, comment: Option[String] = None)
```

Comment:
these's no IR for column detail. use StructField everywhere. `comment` is part of the metadata for a struct field, but it has to be added generically and separately.

Reply:
can we do 
`case class ColumnDetail(structField: StructField, comment: Option[String] = None)` ?

Reply:
ok